### ISSUE TITLE
Multi-gpu issue.

### ISSUE THREAD

**Issue by LifeBeyondExpectations** [Xiaoming-Yu/SingleGAN#4]
Thanks for your contribution of CBIN layer.

I have a short question of lines of code of CBIN. 

In the the `class _CBINorm(_BatchNorm)` of **cbin.py** file, ,
[https://github.com/Xiaoming-Yu/SingleGAN/blob/608551b40fe6107c9e5ce48e27cb8ca29b3b7a8c/models/cbin.py#L31](url)
[https://github.com/Xiaoming-Yu/SingleGAN/blob/608551b40fe6107c9e5ce48e27cb8ca29b3b7a8c/models/cbin.py#L32](url)

The url indicates the line 31 and 32 in the **cbin.py** file.
`self.running_mean.copy_(running_mean.view(b, c).mean(0, keepdim=False))`
`self.running_var.copy_(running_var.view(b, c).mean(0, keepdim=False))`

Why this operation is needed??
I check that for the multi-gpu issue, this raise a runtime error. 
"RuntimeError: one of the variables needed for gradient computation has been modified by an inplace operation"

In short, my question is the necessity of the two lines and the multi-gpu operation issue. 
I currently use torch 0.4.1 and cuda 9.0.

**Comment by Xiaoming-Yu** [comment]
Thanks for your feedback.
The original implementation of CBIN is based on [Instance Normalization](https://pytorch.org/docs/0.3.1/_modules/torch/nn/modules/instancenorm.html) in pytorch 0.3.1.
The running stats buffers seems to have changed in pytorch 0.4.1.
I have refactored CBN based on  pytorch 0.4.1, but I can't make sure this implementation can run in multi-gpu environment.

**Comment by LifeBeyondExpectations** [comment]
Thanks for your feedback. 

[https://github.com/Xiaoming-Yu/SingleGAN/blob/d1435a9644c667e12b7ce4cca0708256e268df36/models/cbin.py#L43](url)
`super(_InstanceNorm, self)._load_from_state_dict(`
The code has the wrong argument. I think that `super(**_CBINorm**, self)._load_from_state_dict(` is correct.

Sorry for the sensitive question, for me, after I change the code with the new one(updated 2018.11.20), I did not get the same results as you published in the papers.

**Comment by Xiaoming-Yu** [comment]
Hi,
I have fix the bug of  wrong argument in CBIN and test it on _apple2orange_(base) and _label2facades_(multimodal).
